internal/integration/helpers.go

```diff
@@ -55,23 +55,32 @@ import (
 func NewTestServer(tb testing.TB) (*serverenv.ServerEnv, *Client) {
 	tb.Helper()
 
-	ctx := context.Background()
-
-	bs, err := storage.NewMemory(ctx)
-	if err != nil {
-		tb.Fatal(err)
+	// Do not start the server when running short tests.
+	if testing.Short() {
+		tb.SkipNow()
 	}
 
-	if v := os.Getenv("GOOGLE_CLOUD_BUCKET"); v != "" && !testing.Short() {
+	ctx := context.Background()
+
+	var bs storage.Blobstore
+	var err error
+	if v := os.Getenv("INTEGRATION_GOOGLE_CLOUD_BUCKET"); v != "" {
+		// Use a real Cloud Storage bucket if this envvar is set.
 		bs, err = storage.NewGoogleCloudStorage(ctx)
 		if err != nil {
 			tb.Fatal(err)
 		}
+	} else {
+		bs, err = storage.NewMemory(ctx)
+		if err != nil {
+			tb.Fatal(err)
+		}
 	}
 
-	db := database.NewTestDatabase(tb)
-	if v := os.Getenv("DB_NAME"); v != "" && !testing.Short() {
-		dbConfig := &database.Config{}
+	var db *database.DB
+	if v := os.Getenv("INTEGRATION_DB_NAME"); v != "" {
```

Comment:
The logic inside this if clause uses `DB_NAME` env var for establishing database connection from e2e test, I would suggest change this if condition to be
```suggestion
	if v := os.Getenv("RUN_E2E_TEST"); v != "" {
```

Reply:
Based on the discussion at https://github.com/google/exposure-notifications-verification-server/pull/841#discussion_r506455010, `INTEGRATION_DB_NAME` would work. Needs several additional changes at:

- Prefix `INTEGRATION_` the `DB_` vars:  https://github.com/google/exposure-notifications-server/blob/5ddd4aa0799a029927eb9e5f74fa245d69e4edad/scripts/e2e-test.sh#L47
- Add  `env:",prefix:INTEGRATION_"` at https://github.com/google/exposure-notifications-server/blob/5ddd4aa0799a029927eb9e5f74fa245d69e4edad/internal/e2e/e2e_test.go#L53